sw/device/lib/testing/usb_testutils_controlep.c

```diff
@@ -436,3 +437,41 @@ status_t usb_testutils_controlep_init(usb_testutils_controlep_ctx_t *ctctx,
 
   return OK_STATUS();
 }
+
+// Proceed only when the device has been configured; this allows host-side
+// software to establish communication.
+status_t usb_testutils_controlep_config_wait(
+    usb_testutils_controlep_ctx_t *ctxctx, usb_testutils_ctx_t *ctx) {
```

Comment:
```suggestion
    usb_testutils_controlep_ctx_t *ctrl_ctx, usb_testutils_ctx_t *ctx) {
```

Reply:
Oh, that's my fault; I was trying to mimic the existing code and naming, but somehow misread `ctctx`, thinking to myself that `ctxctx` is a poor name choice!